docs/api-doc-gen/templates/members.tpl

```diff
@@ -28,18 +28,6 @@
 
         {{ safe (renderComments .CommentLines) }}
 
-        {{- if eq .Type.Kind "Alias" }}
```

Comment:
Instead of removing this, change this line to:
```
        {{- if and (eq .Type.Kind "Alias") (eq .Type.Name.Package "github.com/openshift/hypershift/api/v1alpha1") }}
```